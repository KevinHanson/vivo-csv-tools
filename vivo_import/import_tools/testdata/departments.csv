URI,Department
1000,"Anatomy"
1001,"Anesthesiology"
1002,"Medicine"
