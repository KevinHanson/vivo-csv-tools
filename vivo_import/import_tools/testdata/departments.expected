<http://destination.dartmouth.edu/vivo/individual/org1000> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://vivoweb.org/ontology/core#AcademicDepartment> .
<http://destination.dartmouth.edu/vivo/individual/org1000> <http://www.w3.org/2000/01/rdf-schema#label> "Anatomy"^^<http://www.w3.org/2001/XMLSchema#string> .

<http://destination.dartmouth.edu/vivo/individual/org1001> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://vivoweb.org/ontology/core#AcademicDepartment> .
<http://destination.dartmouth.edu/vivo/individual/org1001> <http://www.w3.org/2000/01/rdf-schema#label> "Anesthesiology"^^<http://www.w3.org/2001/XMLSchema#string> .

